scripts/check_vms_ic_frozen.py

```diff
@@ -0,0 +1,21 @@
+from vcenter import vcenter
+
+vm_to_uuid_dict = {
+    'Win_10_Pro_64b_EN-ss-test_danek-26': '420d4985-a279-1150-2f49-21b336e0e061'
```

Comment:
I would prefer to load this data from command line.

Reply:
Good point, fixed! 👍